crates/uv/tests/common/mod.rs

```diff
@@ -44,7 +44,7 @@ pub const INSTA_FILTERS: &[(&str, &str)] = &[
     (r"uv.exe", "uv"),
     // uv version display
     (
-        r"uv \d+\.\d+\.\d+( \(.*\))?",
+        r"uv(-.*)? \d+\.\d+\.\d+( \(.*\))?",
```

Comment:
We output `uv-run ...` for `uv run --version`. Kind of annoying tbh but it's Clap's behavior I think so just going to filter it.